lib/cc/cli/init.rb

```diff
@@ -18,14 +18,13 @@ def run
       private
 
       def create_codeclimate_yaml
-        config = { "engines" => {} }
-        eligible_engines.each do |engine_name, engine_config|
-          config["engines"][engine_name] = {
-            "enabled" => true
-          }
-          config["ratings"] ||= {}
-          config["ratings"]["paths"] ||= []
+        config = {
+          "engines" => {},
+          "ratings" => { "paths" => [] },
+        }
 
+        eligible_engines.each do |engine_name, engine_config|
+          config["engines"][engine_name] = build_engine_config(engine_config)
```

Comment:
What do you think about doing something like...:

```ruby
def create_codeclimate_yaml
  # ...
  config["engines"][engine_name]["config"] = build_engine_config(engine_config)
  # ...
end
```

...and then just have `build_engine_config` return the appropriate config hash?